src/engine/LocalVocab.h

```diff
@@ -50,60 +54,68 @@ class LocalVocab {
   LocalVocab(const LocalVocab&) = delete;
   LocalVocab& operator=(const LocalVocab&) = delete;
 
-  // Make a logical copy. The clone will have an empty primary set so it can
-  // safely be modified. The contents are copied as shared pointers to const, so
-  // the function runs in linear time in the number of word sets.
+  // Make a logical copy, where all sets of `LocalVocabEntry`s become static
```

Comment:
do you mean `constant` ? 
The only thing that we know, is that the copy won't change them, and that it won't be cleared. The otherWordSets could potentially even grow, but we don't care.